survey-of-agency-tools.md

```diff
@@ -0,0 +1,149 @@
+# Survey of agency performance tools
```

Comment:
Should this be "performance measurement" or "performance analysis" for clarity? (I tried to see if there's a standard term for business intelligence tools like this, but... "business intelligence" seems to be the only standard term.)

Reply:
I'm not going for any specific definition here, it's mean to be generally "any software or systems that agencies are using for performance management, business intelligence, or budget formulation". But I will add this as a definition.